widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -4609,42 +4614,118 @@ div.dijitDateTextBox.dijitComboBox {
   color: #333;
 }
 
-/* added by brett to override browser styles */
+/* region override browser styles */
+
+/* Home and Browser Files perspective */
 .main-container {
   margin: 0 auto;
   max-width: 1230px;
   width: auto;
 }
 
-/* Browse Files */
+/* region Browse Files perspective */
+.browse-files-perspective body {
+  height: 100vh;
+}
+
+.browse-files-perspective.browse-files-perspective :is(.main-container, .main-container > *) {
+  position: unset;
+  padding: 0;
+  height: 100%;
+}
+
+#fileBrowser {
+  display: grid;
+  width: 100%;
+  min-height: 100%;
+
+  grid-template-areas: "folder file actions";
+  grid-template-columns: 1fr 1fr auto;
+  grid-template-rows: 1fr;
+}
+
+/* region .fileBrowserColumn elements  */
+.fileBrowserColumn.fileBrowserColumn.fileBrowserColumn.fileBrowserColumn {
+  min-width: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+}
+
+#fileBrowserFolders {
```

Comment:
Shouldn't these be changed to classes as well if we have created classes similar to ID's